### PR TITLE
Improve site responsiveness on tablet/mobile

### LINKED ISSUE #57: Improve site responsiveness on tablet/mobile
### Feature description

The site works great but isn't overly responsive. We should improve the responsiveness of the site so that it works well on mobile devices as well as tablet devices and different desktop sizes.

### How the feature is useful

Allows more folks to easily access and use the site.

### PR DESCRIPTION
**Pull request by rsales**
## Type of Change

- **Layouts:** I modified the layouts **navbar**, **about page** and **library card** version `mobile/tablet`
- **Components:** Modified the following components `nav.vue` and `library_card.vue`
- **Styling:**  Building different styles to work the different versions of devices.

## What issue does this relate to?

fixes #57 - Improve site responsiveness on tablet/mobile

### What should this PR do?

I adapted some elements of the layout for the mobile version.
* Treatment of styles for different sizes
* Adaptation of the component nav to different sizes

### What are the acceptance criteria?

I believe that the changes are consistent with the request.

## Screenshot of modifications
Screen Shot 2020-10-18 at 00 16 28
Screen Shot 2020-10-18 at 00 16 35
Screen Shot 2020-10-18 at 00 22 09
Screen Shot 2020-10-18 at 00 22 23
Screen Shot 2020-10-18 at 00 22 59

### CODE REVIEW

File: components/nav.vue [hunk 1]
```diff
@@ -34,9 +34,14 @@
                 <a :href="utm('https://github.com/cdnjs', 'nav')">
                     GitHub
                 </a>
-                <a :href="utm('https://status.cdnjs.com', 'nav')">
-                    Status <StatusIndicator page-id="1fkb7yl9sw87"></StatusIndicator>
-                </a>
+                <div class="fix-mobile">
+                    <a href="javascript:void(0);" class="icon-mobile" @click="openMenu">
```
Comment (MattIPv4) [inline components/nav.vue:38]:
```suggestion
                    <a class="icon-mobile" @click="toggleMenu" aria-label="Toggle menu">
```
I don't think we need `href` here + add an `aria-label` for screenreaders + let's call the method toggle.

File: components/nav.vue [hunk 2]
```diff
@@ -53,10 +59,19 @@
...
         },
         methods: {
             utm,
+            openMenu () {
+                const navBar = this.$refs.nav;
+                if (navBar.className === 'nav') {
+                    navBar.className += ' responsive';
+                } else {
+                    navBar.className = 'nav';
+                }
```
Comment (MattIPv4) [inline components/nav.vue:73]:
```suggestion
            toggleMenu () {
                this.$refs.nav.classList.toggle('responsive');
```
I think we can just toggle the class here?

File: assets/scss/components/_nav.scss [hunk 3]
```diff
@@ -2,7 +2,9 @@
   background: $darker;
   box-shadow: $shadow-bottom;
   width: 100%;
-  z-index: 1;
+  z-index: 2;
+  overflow: hidden;
```
Comment (MattIPv4) [inline assets/scss/components/_nav.scss:6]: This breaks the navbar search

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
Instead of overlaying the mobile menu over the content, I wonder if it'd be cleaner to have it just shift the content down below the menu when it folds out? This'll also ensure that if the menu is taller than the screen, the user can easily just scroll the page to access all the menu items?

**Comment by rsales** [comment]
Hi @MattIPv4 , I ended up making the changes you mentioned here in the PR review.

Really to facilitate the user's movement in the mobile version I fixed the menu at the top and when he scrolls the menu will move along with the layout as it was before and when the menu expands it will move the remaining elements of the page.

Thank you for your feedback 🙂

File: components/nav.vue (hunk 1 above)
Comment (rsales) [reply components/nav.vue]: I really liked the solution by removing `javascript:void(0)` inside the `href`

**Review by MattIPv4 — APPROVED** [review]
Lgtm, tyvm!

**Pull request merged**